modules/my-module/android/src/main/java/expo/modules/mymodule/JsRescheduleConfirmationObject.kt

```diff
@@ -0,0 +1,16 @@
+package expo.modules.mymodule
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class JsRescheduleConfirmationObject(
+    val event_id: Long,
```

Comment:
Nit: More of a question than anything. Can we camelcase these and they just work?

Reply:
can comeback for this another time not worth the trouble right now because it works